### ISSUE TITLE
SQL Server Update of AD Admin not working using Azure Cli

### ISSUE THREAD

**Issue by abhiramani-iptiq** [Azure/azure-cli#12768]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Describe the bug**
Both of below commands does not work to update AD Admin on SQL Server.keep getting error
az sql server ad-admin update: error: the following arguments are required: --parameters
usage: az sql server ad-admin update [-h] [--verbose] [--debug]
                                     [--output {json,jsonc,table,tsv,yaml,yamlc,none}]
                                     [--query JMESPATH]
                                     [--subscription _SUBSCRIPTION]
                                     [--resource-group RESOURCE_GROUP_NAME]
                                     [--server-name SERVER_NAME] --parameters
                                     PARAMETERS [--display-name LOGIN]
                                     [--object-id SID]
                                     [--set KEY=VALUE [KEY=VALUE ...]]
                                     [--add LIST KEY=VALUE [LIST KEY=VALUE ...]]
                                     [--remove LIST INDEX [LIST INDEX ...]]
                                     [--force-string] [--ids ID [ID ...]]

**To Reproduce**
az sql server ad-admin update --id value --object-id value 

az sql server ad-admin update --server-name value  --resource-group value --subscription value  --object-id value 

**Expected behavior**
above commands should update my SQL AD Admin

**Environment summary**
Azure Cloud Shell

**Additional context**

**Comment by yonzhan** [comment]
sql

**Comment by jaredmoo** [comment]
Fixed by #12541. Thanks!

**Comment by abhiramani-iptiq** [comment]
@jaredmoo How is this fixed? i am still getting error  and still not working in Azure cloud shell

**Comment by jaredmoo** [comment]
Running `az version` in Azure cloud shell today:

```
> az version
This command is in preview. It may be changed/removed in a future release.
{
  "azure-cli": "2.2.0",
  "azure-cli-command-modules-nspkg": "2.0.3",
  "azure-cli-core": "2.2.0",
  "azure-cli-nspkg": "3.0.4",
  "azure-cli-telemetry": "1.0.4",
  "extensions": {}
}
```

At https://pypi.org/project/azure-cli/2.2.0/ you can see that 2.2.0 was released on March 9, which is before #12541 was merged on March 16. So the fix is just not yet published. Cheers :)
